### DISCUSSION TITLE
New Evennia devblog!

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2584] [Evennia Official]
https://www.evennia.com/devblog

Not only a new dev blog, but a whole new blog platform! I have migrated from blogspot/bloggly to use a custom solution of my own on evennia.com. Read all about it. 

This thread can be used to discuss/ask questions about dev blogs if you have any. :)

**Comment by talljosh** [comment]
I'm going to miss having an RSS feed of the dev blog. At present the blog's the only part of Evennia I follow, so the reality is that unless there's some kind of notification channel dedicated only to dev blog posts, I'll probably miss out on reading the posts now.

**Reply by Griatch** [reply]
@talljosh Ok, I added an RSS feed to the new blog now, enjoy and hope to see you follow more news than just the blog in the future :slightly_smiling_face:

**Reply by talljosh** [reply]
Much appreciated, thanks!

**Comment by Griatch** [comment]
New dev blog: https://www.evennia.com/devblog/2022.html#2022-01-06-into-2022-with-thanks-and-plans

**Comment by Griatch** [comment]
Another Evennia Dev blog! 
https://www.evennia.com/devblog/2022.html#2022-07-05-tutorial-writing-and-attributes-galore

**Comment by Griatch** [comment]
New Evennia dev blog Sept 17, 2022: https://www.evennia.com/devblog/2022.html#2022-09-17-project-plans-and-splitting-a-setting-in-two

**Comment by Griatch** [comment]
And a new Devblog to talk about the release and the future: 
https://www.evennia.com/devblog/2022.html#2022-12-03-evennia-1.0-released!

**Comment by Griatch** [comment]
A new Devblog talking about the latest 2.0.0 release: 
https://discord.com/channels/246323978879107073/246323978879107073/1117126794529742858
